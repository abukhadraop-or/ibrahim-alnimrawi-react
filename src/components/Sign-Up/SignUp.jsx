import React from "react";

/**
 * Render a <SignUp> component
 */
const SignUp = () => {
  return <div>this is sign up</div>;
};

export default SignUp;
