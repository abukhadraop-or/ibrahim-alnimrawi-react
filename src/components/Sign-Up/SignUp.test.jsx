import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Enzyme from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import SignUp from './SignUp';

Enzyme.configure({ adapter: new Adapter() });

describe('<SignUp />', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<SignUp />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
