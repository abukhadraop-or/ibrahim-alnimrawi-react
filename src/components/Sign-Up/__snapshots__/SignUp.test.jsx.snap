// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUp /> should match the snapshot 1`] = `
<div>
  this is sign up
</div>
`;

exports[`<SignUp /> should match the snapshot be 1`] = `
<div>
  this is sign up
</div>
`;
